src/_components/NavBarComponent.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import jwt from 'jsonwebtoken';
+import { Dropdown, Menu, Icon } from 'semantic-ui-react';
+
+import '../_css/NavBarComponent.css';
+
+const NavBarComponent = (props) => {
```

Comment:
- Select everything inside `NavBarComponent` and move back by 2 spaces. If you are using visual studio, then the command to use after selecting is `cmd + [`. 
- You should also set your editor to use **_2 spaces_** for js/jsx files